src/packs.rs

```diff
@@ -49,6 +49,65 @@ pub fn greet() {
     println!("👋 Hello! Welcome to packs 📦 🔥 🎉 🌈. This tool is under construction.")
 }
 
+
+pub fn init(absolute_root: &PathBuf) -> anyhow::Result<()> {
+    let root_package = "\
+# This file represents the root package of the application
+# Please validate the configuration using `packwerk validate` (for Rails applications) or running the auto generated
+# test case (for non-Rails projects). You can then use `packwerk check` to check your code.
+
+# Change to `true` to turn on dependency checks for this package
+enforce_dependencies: false
+
+# A list of this package's dependencies
+# Note that packages in this list require their own `package.yml` file
+# dependencies:
+# - \"packages/billing\"
+";
+    let packs_config = "\
+# See: Setting up the configuration file
+# https://github.com/Shopify/packwerk/blob/main/USAGE.md#configuring-packwerk
+
+# List of patterns for folder paths to include
+# include:
+# - \"**/*.{rb,rake,erb}\"
+
+# List of patterns for folder paths to exclude
+# exclude:
+# - \"{bin,node_modules,script,tmp,vendor}/**/*\"
+
+# Patterns to find package configuration files
+# package_paths: \"**/\"
+
+# List of custom associations, if any
+# custom_associations:
+# - \"cache_belongs_to\"
+
+# Whether or not you want the cache enabled (disabled by default)
+# cache: true
+
+# Where you want the cache to be stored (default below)
+# cache_directory: \"tmp/cache/packwerk\"
+";
+    let root_package_path = absolute_root.join("package.yml");
+    let packs_config_path = absolute_root.join("packwerk.yml"); // Can you just create a packs.yml?
```

Comment:
I think we should actually only write one for now... I'm thinking just writing `packwerk.yml` makes sense, in the interest of this gem being a strict port of `packwerk` to start?

On the other hand, I can easily see the argument that anyone using `init` with `pks` maybe should just begin in `packs_first_mode` from the beginning, what do you think?

Reply:
I think it prob makes sense for `pks init` to generate `packs.yml`, unless a switch says othewise as you mentioned in your other comment. Would be happy to drop a CLI switch in if I can work it out